src/LlmTornado.Agents/LlmTornado.Agents.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Lombda.StateMachine" Version="1.0.1" />
```

Comment:
Can we move this into Agents codebase, possibly a standalone project LlmTornado.StateMachine so we don't have external dependencies?